internal/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -208,6 +208,10 @@ type CertificateSpec struct {
 	// +kubebuilder:validation:ExclusiveMaximum=false
 	// +optional
```

Comment:
Guess this branch needs to be rebased since `v1alpha2` doesn't exist anymore?

Reply:
I think it shouldn't exists anymore but it's still present on the master branch.
If we run the `./hack/update-codegen.sh` script from the master branch it remove a lot of conversion / deepcopy functions (and it doe not build anymore).
That's not the goal of this branch.

Reply:
Thanks @seuf and sorry for the confusion here. I had not appreciated the fact that the old APIs had moved under `internal`, rather than being deleted completely.